tmp/rubycritic/app/frame.html

```diff
@@ -0,0 +1,160 @@
+<!DOCTYPE html>
```

Comment:
Not sure if these `tmp` files are needed.
If not, you can ignore them by adding `tmp/` in your `.gitignore`

Reply:
You're right, they aren't! Thanks!